apps/omg_child_chain/lib/child_chain.ex

```diff
@@ -27,11 +27,10 @@ defmodule OMG.ChildChain do
   alias OMG.State
   alias OMG.State.Transaction
   use OMG.Utils.LoggerExt
-  use OMG.Utils.Metrics
+  use Spandex.Decorators
```

Comment:
I commented once on this before (in another file), but are we decorating anything in this module? Does this need to be here?

Reply:
nop! leftover!